base/event/FairMCEventHeader.cxx

```diff
@@ -19,7 +19,7 @@
 FairMCEventHeader::FairMCEventHeader()
     : TNamed("MCEvent", "MC")
     , fRunId(0)
-    , fEventId(-1)
+    , fEventId(NoEventID)
```

Comment:
If you touch this member variable anyhow, could you move this to a member initializer?

Reply:
that is a member initializer :P you mean default member intialization? done